serving/docs/configurations.md

```diff
@@ -0,0 +1,177 @@
+|MODEL_SERVER_HOME	|env var	| DJLServing home directory, default: Installation directory (e.g. /usr/local/Cellar/djl-serving/0.19.0/)	                                                                                                                                                                                                                                                                                                                                                  |
+|DEFAULT_JVM_OPTS	|env var	| default: `-Dlog4j.configurationFile=${APP_HOME}/conf/log4j2.xml`<br>Override default JVM startup options and system properties.                                                                                                                                                                                                                                                                                                                           
+|JAVA_OPTS         	|env var   	| default: `-XX:+UseContainerSupport -XX:+ExitOnOutOfMemoryError`<br>Add extra JVM options.                                                                                                                                                                                                                                                                                                                                                                 
+|SERVING_OPTS      	|env var   	| default: N/A<br>Add serving related JVM options.<br>Some of DJL configuration can only be configured by JVM system properties, user has to set DEFAULT_JVM_OPTS environment variable to configure them.<br>- `-Dai.djl.pytorch.num_interop_threads=2`, this will override interop threads for PyTorch<br>- `-Dai.djl.pytorch.num_threads=2`, this will override OMP_NUM_THREADS for PyTorch<br>- `-Dai.djl.logging.level=debug` change DJL loggging level 
+
+## Model specific settings
+
+You set per model settings by adding a `serving.properties` file in the root of your model directory (or .zip).
+
+You can set number of workers for each model:
+https://github.com/deepjavalibrary/djl-serving/blob/master/serving/src/test/resources/identity/serving.properties#L4-L8
+
+For example, set minimum workers and maximum workers for your model:
+
+```
+minWorkers=32
+gmaxWorkers=64
+```
+
+Or you can configure minimum workers and maximum workers differently for GPU and CPU:
+
+```
+gpu.minWorkers=2
+gpu.maxWorkers=3
+cpu.minWorkers=2
+cpu.maxWorkers=4
+```
+
+job queue size can be configured at per model level, this will override global `job_queue_size`:
+
+```
+job_queue_size=10
+```
+
+### Rubikon (DeepSpeed)
+
+For Rubikon (DeepSpeed) engine, DJL load multiple workers sequentially by default to avoid run out of memory. You can reduced model loading time by parallel loading workers if you know the peak memory won’t cause out of memory:
```

Comment:
```suggestion
For Python (DeepSpeed) engine, DJL load multiple workers sequentially by default to avoid run out of memory. You can reduced model loading time by parallel loading workers if you know the peak memory won’t cause out of memory:
```